vstools/enums/other.py

```diff
@@ -72,9 +72,9 @@ def from_size(
         sar: Sar | Literal[False]
 
         if isinstance(clip_width, vs.VideoNode):
-            from ..functions.check import check_variable_resolution
+            from ..functions import check_variable_resolution
 
-            check_variable_resolution(clip_width, cls.from_size)  # type:ignore
+            assert check_variable_resolution(clip_width, cls.from_size)  # type:ignore
```

Comment:
assert isn't really needed for variable resolution, the typeguard doesn't do anything

Reply:
remnant of when I had `check_variable` there before realising I only had to check the res